python/cudf_polars/cudf_polars/dsl/ir.py

```diff
@@ -1315,10 +1314,15 @@ def do_evaluate(
             )
             if coalesce and how != "inner":
                 left = left.with_columns(
-                    (
+                    tuple(
```

Comment:
Was this a generator before? That seems surprising that it didn't cause any errors. Is it OK for the generator to be exhausted?